src/helpers.cc

```diff
@@ -266,11 +273,17 @@ LiteClient::LiteClient(Config& config_in)
   primary_ecu = ecu_serials[0];
 
   std::vector<std::string> headers;
-  GObjectUniquePtr<OstreeSysroot> sysroot_smart = OstreeManager::LoadSysroot(config.pacman.sysroot);
-  OstreeDeployment* deployment = ostree_sysroot_get_booted_deployment(sysroot_smart.get());
+  bool booted_sysroot = true;
+  if (config.pacman.extra.count("booted") == 1) {
+    booted_sysroot = boost::lexical_cast<bool>(config.pacman.extra.at("booted"));
```

Comment:
neat. i do some boolean stuff on config elsewhere that could use this boost helper.